### PR TITLE
Update to Web Editor wiki section

### PR DESCRIPTION
**Pull request by Laarryy**
Remade some shiny new gifs, added the nodes the players can use to modify Luckperms things (`group.<group>` etc.) and added that multiple permissions can be added at once, as well as how to edit contexts more clearly.

### CODE REVIEW

**Comment by Andre601** [comment]
#35

**Comment by Laarryy** [comment]
mine's better :P 
only cause there's a bunch of stuff extra, yours is almost the same but without like 1 table and 3 paragraphs xD

EDIT: also only saw it after I PRd, i probably would've PRd your PR if I knew. sowwy :cry:

**Comment by Laarryy** [comment]
added most of the changes from #35 cause they were good

**Comment by Andre601** [comment]
My point is not which one is better, but that you should suggest changes to already existing PRs rather than make a separate one.

**Comment by Laarryy** [comment]
you're right, and it's my fault for not seeing it - im sorry.

**Comment by Andre601** [comment]
Also, knowing @Lucko would he prioritize mine, because it was made first.

**Comment by lucko** [comment]
Thanks :)

> Also, knowing @lucko would he prioritize mine, because it was made first.

I've merged both of them.

File: Web-Editor.md [hunk 1]
```diff
@@ -37,42 +37,57 @@ ___
...
 
-![](https://thumbs.gfycat.com/SkeletalHarmoniousFattaileddunnart-size_restricted.gif)
+![](https://i.imgur.com/yVN4Lzp.gif)
 
-#### Sorting the data
+#### Luckperms Nodes
 
-To change the sorting settings, click on the column heading you'd like to sort by.
+The editor allows you to add or change aspects abouts groups and players such as weight and parents using nodes.
+This is all of them, and what they do:
```
Comment (Andre601) [inline Web-Editor.md:71]: Bit late, but wouldn't it sound better as `This is a list of all nodes and what they do:`
That grammar is really weird to understand.

**Comment by BrainStone** [comment]
Indeed a bit late, don't you think?

**Comment by Andre601** [comment]
Who expects Luck to merge it that quick?

**Comment by BrainStone** [comment]
Considering his recent fast merges, that wasn't a surprise.

File: Web-Editor.md (hunk 1 above)
Comment (lucko) [reply Web-Editor.md:71]: The current wording is fine imo.

Comment (Andre601) [reply Web-Editor.md:71]: If you say so...

**Pull request merged**